src/main/java/DaveException.java

```diff
@@ -0,0 +1,13 @@
+public class DaveException extends Exception {
+
+    private String error;
+
+    public DaveException(String error) {
+        this.error = error;
+    }
+
+    @Override
+    public String toString() {
+        return this.error;
+    }
+}
```

Comment:
You may want to have a package to house all of your exceptions. I feel use of packages would make the organisation easier to understand.